test/mappings/FormattersSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Please move this file to forms.mappings and once it is moved, remove this redundant mappings package